circle.yml

```diff
@@ -3,9 +3,9 @@ machine:
     version: oraclejdk8
   environment:
     QEMU_AUDIO_DRV: none
-    ANDROID_SDK: "android-25"
-    BUILD_TOOLS: "25.0.3"
-    ANDROID_EMULATOR: "circleci-android24"
+    ANDROID_SDK: "android-26"
+    BUILD_TOOLS: "26.0.1"
+    ANDROID_EMULATOR: "circleci-android25"
```

Comment:
Google did not release armeabi system-image for Android API level 24 and 26 that is why we still using 24 ;)